bin/picca_deltas.py

```diff
@@ -295,6 +302,21 @@ def cont_fit(data):
                         nb_dla_in_forest += 1
         log.write("Found {} DLAs in forests\n".format(nb_dla_in_forest))
 
+    ## Mask BALs
+    if not args.bal is None:
+        print("INFO: Adding BALs found in BAL catalog")
+        bcat = bal_tools.read_bal(args.bal)
+        nb_bal_in_forest = 0
+        for p in data:
+            for d in data[p]:
+                if d.thid in bcat["THING_ID"]:
+                    BAL_mask_obs = []
+                    BAL_mask_rf = bal_tools.add_bal_rf(bcat,d.thid,args.BALi)
```

Comment:
lower case for variable names as well

Reply:
Variable names in both picca_deltas and py/picca/bal_tools.py should be lowercase now.